grails-app/conf/UrlMappings.groovy

```diff
@@ -256,9 +256,9 @@ class UrlMappings {
         }
 
         // Invoice API
-        "/api/invoices/$id/invoiceItems"(parseRequest: true) {
+        "/api/invoices/$id/items"(parseRequest: true) {
```

Comment:
Interesting chose. Can/should we apply this to other APIs in the near future?

Reply:
I think we should if we can specify the same endpoint for different request methods. In this case we had two separate endpoints for no reason.